### PR TITLE
multiple content updates, product nav update

### PR DESCRIPTION
**Pull request by a-chappelear**
**For staging:**
1. Multiple content changes + an image addition to the "product" section
2. Content updates to "Resources"
3. Content updates to "Support" and "Contact"
4. "About" added to the "Product" subnav
5. Homepage content update (removing "Covid")

### Testing
- [x] Tested locally?
- [ ] Ran `./prime test` or `./gradlew testSmoke` against local Docker ReportStream container?
- [x] (For Changes to /frontend-react/...) Ran `npm run lint:write`? 
- [ ] Added tests?

### Process
- [ ] Are there licensing issues with any new dependencies introduced? n/a
- [ ] Includes a summary of what a code reviewer should test/verify? Content (Berni) 
- [ ] Updated the release notes? n/a
- [ ] Database changes are submitted as a separate PR? n/a
- [ ] DevOps team has been notified if PR requires ops support? n/a

Homepage_updates
Contect_updates
Support_updates
Resources_updates
Product_updates

### CODE REVIEW

File: frontend-react/src/pages/support/Faq.tsx [hunk 1]
```diff
@@ -68,60 +78,45 @@ export const Faq = () => {
...
-                    SimpleReport
-                </a>
-                , a PRIME project that ReportStream closely collaborates with.
-                To access your user account, be sure to log in at{" "}
-                <Link to="/login" key="login" className="usa-link">
+                to SimpleReport. To access your ReportStream user account, be
+                sure to log in at{" "}
+                <a
+                    className="usa-link"
+                    href="http://reportstream.cdc.gov/login"
```
Comment (kevinhaube) [inline frontend-react/src/pages/support/Faq.tsx:102]: This is throwing a SonarCloud security alert -- `http` needs to be `https`

**Review by kevinhaube — CHANGES_REQUESTED** [review]

File: frontend-react/src/pages/support/Faq.tsx (hunk 1 above)
Comment (a-chappelear) [reply frontend-react/src/pages/support/Faq.tsx]: This has been fixed and pushed up!

**Review by kevinhaube — COMMENTED** [review]
Code looks good! Leaving content to Berni :)

**Review by kevinhaube — APPROVED** [review]
RE: standup, merging this!

**Pull request merged**